sql/moz-fx-data-shared-prod/jira_service_desk/dataset_metadata.yaml

```diff
@@ -4,7 +4,13 @@ description: |-
 dataset_base_acl: view
 user_facing: true
 labels: {}
+default_table_workgroup_access:
```

Comment:
You should never need to set `default_table_workgroup_access` in the `main` branch. Didn't we add CI to enforce this?

Reply:
I'm thinking of https://github.com/mozilla/bigquery-etl/pull/6822#issuecomment-2596474076. Looks like that PR hasn't landed and the suggestion about adding a check to get rid of `default_table_workgroup_access` hasn't been implemented

Reply:
It hasn't been merged yet: https://github.com/mozilla/bigquery-etl/pull/6822